### PR TITLE
chore(connectors): release new ootbs for 8.0

### PR DESCRIPTION
**Pull request by MaxTru**
## What is the purpose of the change

3 new OOTB connectors will be made available in web modeler today. Hence they are effectively enabled for 8.0.

## Are there related marketing activities

no. Will be marketed as part of CCon.

## When should this change go live?

asap

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by igpetrov — DISMISSED** [review]

**Comment by igpetrov** [comment]
I think, you forgot to move screenshots / images

File: versioned_sidebars/version-8.0-sidebars.json [hunk 1]
```diff
@@ -253,6 +253,9 @@
             {
               "Out-of-the-box Connectors": [
                 "components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview",
+                "components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs",
+                "components/integration-framework/connectors/out-of-the-box-connectors/googledrive",
+                "components/integration-framework/connectors/out-of-the-box-connectors/aws-lambda",
```
Comment (PreetiNirwal) [inline versioned_sidebars/version-8.0-sidebars.json:258]: I'm not sure if this is the area, but... having the Connectors alphabetized on the side panel isn't there in "Next". Could we update this PR to include the proper order: 
1. Amazon SQS Connector
2. AWS Lambda Connector
3. Google Drive Connector
4. REST Connector
5. SendGrid Connector
6. Slack Connector

cc @MaxTru @igpetrov

Comment (MaxTru) [reply versioned_sidebars/version-8.0-sidebars.json:257]: Right. Thanks. Corrected

**Comment by MaxTru** [comment]
Thanks for review @PreetiNirwal and @igpetrov . Adressed your comments

**Review by PreetiNirwal — APPROVED** [review]
Perfect - thanks!

**Review by igpetrov — APPROVED** [review]

**Comment by MaxTru** [comment]
@akeller @christinaausley this is ready from Connectors side. Would only need final docs approval. Then this can be merged. Thanks

File: versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ description: Take a closer look at the Connectors available in Camunda Platform
 
 This section gives an overview of the **out-of-the-box Connectors** available in Camunda Platform 8:
 
+- [Amazon SQS Connector](aws-sqs.md) - Send messages to Amazon [Simple Queue Service](https://aws.amazon.com/sqs/) from your BPMN process.
```
Comment (akeller) [inline versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md:9]:
```suggestion
- [Amazon SQS Connector](aws-sqs.md) - Send messages to [Amazon Simple Queue Service](https://aws.amazon.com/sqs/) from your BPMN process.
```

Small nitpick here.

File: versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+id: aws-sqs
+title: Amazon Simple Queue Service Connector
+sidebar_label: Amazon SQS Connector
+description: Send messages to Amazon Simple Queue Service from your BPMN process.
+---
+
+The Amazon SQS Connector allows you to connect your BPMN service with Amazon Web Service's [Simple Queue Service](https://aws.amazon.com/sqs/).
```
Comment (akeller) [inline versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs.md:8]:
```suggestion
---
id: aws-sqs
title: Amazon Simple Queue Service Connector
sidebar_label: Amazon SQS Connector
description: Send messages to Amazon Simple Queue Service (SQS) from your BPMN process.
---

The Amazon SQS Connector allows you to connect your BPMN service with [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/).
```

Looking at how Amazon talks about this product on their [product page](https://aws.amazon.com/sqs/), I made the above suggestions.

**Review by akeller — APPROVED** [review]
I left a few comments, but I would consider these non-blocking.

Do these changes need to be available in all future versions? If so, we'll need to add them to `/docs/` aws well. They are currently only in `8.0`.

**Pull request merged**